product_code/shared/client/src/main/scala/de/upb/cs/uc4/shared/client/exceptions/UC4Exception.scala

```diff
@@ -57,4 +59,7 @@ object UC4Exception {
 
   def InternalServerError(throwable: Throwable, invalidParams: SimpleError*) =
     new UC4CriticalException(500, DetailedError(ErrorType.InternalServer, invalidParams), throwable)
+
+  //501
+  val NotImplemented = new UC4CriticalException(501, GenericError(ErrorType.NotImplemented), null)
```

Comment:
Is this realy a critical Exception?
This could be used as a DOS atack.

Reply:
Good point, it is now a non-critical exception